<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectFrontendClass</class>
 <widget class="QMainWindow" name="ProjectFrontendClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>餐厅系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbLogin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>500</width>
      <height>225</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QPushButton" name="btnSignUp">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSignIn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>登录系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPhone">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>50</y>
       <width>155</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入您的电话号码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPW">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>100</y>
       <width>155</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入您的密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePhone">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
     <property name="placeholderText">
      <string>11位手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePW">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>8位密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblLoginBGN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lblLoginBGN</zorder>
   <zorder>gbLogin</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProjectFrontend.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSignUp</sender>
   <signal>clicked()</signal>
   <receiver>ProjectFrontendClass</receiver>
   <slot>OnRegClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProjectFrontendClass</receiver>
   <slot>CheckPhone(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePW</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProjectFrontendClass</receiver>
   <slot>CheckPW(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSignIn</sender>
   <signal>clicked()</signal>
   <receiver>ProjectFrontendClass</receiver>
   <slot>OnLogClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePW</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSignIn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSignIn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnRegClicked()</slot>
  <slot>CheckPhone(QString)</slot>
  <slot>CheckPW(QString)</slot>
  <slot>OnLogClicked()</slot>
 </slots>
</ui>
